analysis/data_utils.py

```diff
@@ -128,17 +129,37 @@ def filter_max_time(experiment_df, max_time):
     return experiment_df[experiment_df['time'] <= max_time]
 
 
+def is_unique_crash(crash_group):
+    """Check if each crash in |crash_group| is unique with CF's crash comparer.
+    Return the |crash_group| with an extra columns representing if that crash
+    is the first occurrence."""
+    unique_crashes = set()
+    is_firsts = []
+    for crash in crash_group.crash_key:
```

Comment:
`crash_key` is the concatenation of both the crash type and the crash state: https://github.com/google/fuzzbench/blob/e2761292572de489049aa64d518044f660578f59/experiment/measurer/run_crashes.py#L101

This is inconsistent with CF's grouping behaviour, which does this based only on the crash state (assuming the security relevance is the same). 

We should do a split on ":" here to only consider the crash state. This doesn't take into account the security relevance, but I think that's fine for the purposes of benchmarking.